lib/registries/addon/drafts/draft/draft-registration-manager.ts

```diff
@@ -45,6 +44,8 @@ export default class DraftRegistrationManager {
     schemaBlocks!: SchemaBlock[];
 
     @alias('draftRegistration.id') draftId!: string;
+    @alias('draftRegistration.currentUserIsReadOnly') readOnly!: boolean;
```

Comment:
So the alias below is `currentUserIsAdmin` but this one is `readOnly`? Doesn't seem consistent.